<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1025</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" rowspan="2">
     <widget class="QGroupBox" name="groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Utilisateur</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QComboBox" name="cmb_UserName">
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="editable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Passeports en poche</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QCheckBox" name="chk_PassI3D">
            <property name="text">
             <string>Impression 3D</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="chk_Las">
            <property name="text">
             <string>Laser</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="chk_Plo">
            <property name="text">
             <string>Plotter</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="chk_Sho">
            <property name="text">
             <string>ShopBot</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="chk_Cer">
            <property name="text">
             <string>Céramique</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Informations utilisateur</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Prénom</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="ln_UserName">
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="frame">
             <bool>true</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="readOnly">
             <bool>false</bool>
            </property>
            <property name="clearButtonEnabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Nom</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="ln_LastName">
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Mail</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="ln_Mail">
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Téléphone</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="ln_Phone">
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="4" column="0" colspan="2">
           <widget class="QCheckBox" name="chk_PriceCat">
            <property name="text">
             <string>Tarif Réduit</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1" rowspan="2">
     <widget class="QGroupBox" name="groupBox_5">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Utilisation et Achats</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="CounterDisplay" name="dbl_Quantity">
         <property name="font">
          <font>
           <pointsize>30</pointsize>
          </font>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <double>-99.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.250000000000000</double>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QCommandLinkButton" name="btn_Validate">
         <property name="text">
          <string>Valider</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QComboBox" name="cmb_Prod">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="currentIndex">
          <number>-1</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_11">
           <property name="font">
            <font>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Commentaire</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ln_Comment">
           <property name="font">
            <font>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="2">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="3">
     <widget class="QCheckBox" name="chk_UserFilter">
      <property name="text">
       <string>Activité utilisateur</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Compteurs d'heures</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="1">
        <widget class="CounterDisplay" name="tm_I3D">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="font">
          <font>
           <pointsize>30</pointsize>
          </font>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="minimum">
          <double>-99.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="CounterDisplay" name="tm_Las">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="font">
          <font>
           <pointsize>30</pointsize>
          </font>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="showGroupSeparator" stdset="0">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <double>-99.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lbl_Las">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="font">
          <font>
           <pointsize>20</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Laser</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lbl_I3D">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="font">
          <font>
           <pointsize>20</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>I3D</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lbl_Plo">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="font">
          <font>
           <pointsize>20</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Plotter</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="CounterDisplay" name="tm_Plo">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="font">
          <font>
           <pointsize>30</pointsize>
          </font>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="minimum">
          <double>-99.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="lbl_Sho">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="font">
          <font>
           <pointsize>20</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Shopbot</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="CounterDisplay" name="tm_Sho">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="font">
          <font>
           <pointsize>30</pointsize>
          </font>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="minimum">
          <double>-99.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="lbl_Cer">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="font">
          <font>
           <pointsize>20</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Céramique</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="CounterDisplay" name="tm_Cer">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="font">
          <font>
           <pointsize>30</pointsize>
          </font>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="minimum">
          <double>-99.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="0">
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>38</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="2" rowspan="3" colspan="2">
     <widget class="QTableView" name="lst_Activity">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>0</height>
       </size>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="autoScroll">
       <bool>false</bool>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1025</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionModifications_avanc_es"/>
    <addaction name="separator"/>
    <addaction name="actionPr_f_rences"/>
    <addaction name="actionQuitter"/>
   </widget>
   <addaction name="menuFichier"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionModifications_avanc_es">
   <property name="text">
    <string>Modifications avancées</string>
   </property>
  </action>
  <action name="actionPr_f_rences">
   <property name="text">
    <string>Préférences</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CounterDisplay</class>
   <extends>QDoubleSpinBox</extends>
   <header>widgets/CounterDisplay.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cmb_UserName</tabstop>
  <tabstop>ln_UserName</tabstop>
  <tabstop>ln_LastName</tabstop>
  <tabstop>ln_Mail</tabstop>
  <tabstop>ln_Phone</tabstop>
  <tabstop>chk_PriceCat</tabstop>
  <tabstop>chk_PassI3D</tabstop>
  <tabstop>chk_Las</tabstop>
  <tabstop>chk_Plo</tabstop>
  <tabstop>chk_Sho</tabstop>
  <tabstop>chk_Cer</tabstop>
  <tabstop>dbl_Quantity</tabstop>
  <tabstop>cmb_Prod</tabstop>
  <tabstop>ln_Comment</tabstop>
  <tabstop>btn_Validate</tabstop>
  <tabstop>tm_I3D</tabstop>
  <tabstop>tm_Las</tabstop>
  <tabstop>tm_Plo</tabstop>
  <tabstop>tm_Sho</tabstop>
  <tabstop>tm_Cer</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>chk_Plo</sender>
   <signal>toggled(bool)</signal>
   <receiver>lbl_Plo</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>346</x>
     <y>204</y>
    </hint>
    <hint type="destinationlabel">
     <x>81</x>
     <y>455</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chk_Sho</sender>
   <signal>toggled(bool)</signal>
   <receiver>tm_Sho</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>263</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>519</x>
     <y>343</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chk_Cer</sender>
   <signal>toggled(bool)</signal>
   <receiver>tm_Cer</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>310</x>
     <y>249</y>
    </hint>
    <hint type="destinationlabel">
     <x>529</x>
     <y>409</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chk_UserFilter</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>UserFilter(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>944</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>744</x>
     <y>56</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chk_Las</sender>
   <signal>toggled(bool)</signal>
   <receiver>lbl_Las</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>346</x>
     <y>182</y>
    </hint>
    <hint type="destinationlabel">
     <x>79</x>
     <y>396</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_Validate</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>ValidateEntry()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>586</x>
     <y>263</y>
    </hint>
    <hint type="destinationlabel">
     <x>743</x>
     <y>472</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chk_PassI3D</sender>
   <signal>toggled(bool)</signal>
   <receiver>tm_I3D</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>304</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>329</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chk_Las</sender>
   <signal>toggled(bool)</signal>
   <receiver>tm_Las</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>289</x>
     <y>182</y>
    </hint>
    <hint type="destinationlabel">
     <x>207</x>
     <y>390</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chk_Sho</sender>
   <signal>toggled(bool)</signal>
   <receiver>lbl_Sho</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>344</x>
     <y>229</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>331</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chk_PassI3D</sender>
   <signal>toggled(bool)</signal>
   <receiver>lbl_I3D</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>346</x>
     <y>154</y>
    </hint>
    <hint type="destinationlabel">
     <x>106</x>
     <y>332</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chk_Cer</sender>
   <signal>toggled(bool)</signal>
   <receiver>lbl_Cer</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>289</x>
     <y>251</y>
    </hint>
    <hint type="destinationlabel">
     <x>360</x>
     <y>400</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chk_Plo</sender>
   <signal>toggled(bool)</signal>
   <receiver>tm_Plo</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>202</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>455</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmb_UserName</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>UserChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>179</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>396</x>
     <y>30</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>ValidateEntry()</slot>
  <slot>UserChanged()</slot>
  <slot>UserFilter(bool)</slot>
 </slots>
</ui>
